### ISSUE TITLE
Enforce BIDS Derivatives-compatible ICA inputs

### ISSUE THREAD

**Issue by tsalo** [ME-ICA/aroma#17]
We should restructure the ICA outputs that ICA-AROMA ingests to be BIDS Derivatives-compatible. ~~In order to maintain support for MELODIC, I think we need a new function that converts MELODIC outputs to a BIDS Derivatives outputs.~~ The main ICA-AROMA functions would then just use BIDS Derivatives outputs.

**Comment by oesteban** [comment]
Totally in favor of this.

> I think we need a new function that converts MELODIC outputs to a BIDS Derivatives outputs. The main ICA-AROMA functions would then just use BIDS Derivatives outputs.

Yes, but let's not make that function part of this issue and give it a super-low priority.

**Comment by oesteban** [comment]
In other words, I would replace "Accept" with "Enforce" in the title of this issue, and assume that the input components are BIDS compliant.

**Comment by tsalo** [comment]
That works for me. I think a MELODIC-to-BIDS Derivatives function should go in a separate repository anyway. I would just prefer to have it as a Python function if at all possible, so that AROMA users won't need to get `nipype` working to use it.

**Comment by oesteban** [comment]
> I would just prefer to have it as a Python function if at all possible

Makes sense. If this project progresses quickly, that could be a nice feature to have 👍
